ocaml/idl/datamodel.ml

```diff
@@ -9200,6 +9202,50 @@ let alert =
     ()
 *)
 
+let net_sriov_create = call
+    ~name:"create"
+    ~doc:"Create a net-sriov on specified pif"
+    ~params:[Ref _pif, "pif", "PIF which to enable sriov";
```

Comment:
`PIF {on/for} which to enable sriov`. Maybe `SRIOV` would be more consistent?

Reply:
Thanks!